src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/BasicHtmlWebResponseObject.Common.cs

```diff
@@ -167,11 +167,6 @@ protected void InitializeContent()
                 // Fill the Content buffer
                 string characterSet = WebResponseHelper.GetCharacterSet(BaseResponse);
 
-                if (string.IsNullOrEmpty(characterSet) && ContentHelper.IsJson(contentType))
-                {
-                    characterSet = Encoding.UTF8.HeaderName;
-                }
```

Comment:
Why is this removed? I don't see this code moved to `StreamHelper.DecodeStream`.